ui/shared/components/panel/view-fields/BaseFieldView.jsx

```diff
@@ -199,6 +200,7 @@ class BaseFieldView extends React.PureComponent {
           content="Only visible to internal users."
         />
       ),
+      isRequired && <Icon key="required" name="asterisk" size="small" />,
```

Comment:
You may want to hide the asterisk when the field has a value.

Reply:
no per the ticket we always want to show its required